Resources/config/server.xml

```diff
@@ -30,7 +30,8 @@
         <service id="ftrrtf_rollbar.rollbar_listener" class="%ftrrtf_rollbar.rollbar_listener.class%">
             <argument type="service" id="ftrrtf_rollbar.notifier"/>
             <argument type="service" id="ftrrtf_rollbar.error_handler"/>
-            <argument type="service" id="security.context"/>
+            <argument type="service" id="security.token_storage"/>
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
And the same thing for `security.authorization_checker` and `security.token_storage`, they are supported from version 2.6+ only